### DISCUSSION TITLE
<iostream> cannot be found.

### DISCUSSION THREAD

**Discussion by ghost** [microsoft/vscode-cpptools#9903] [Q&A]
I am quite new to coding with C++ and I am trying to begin on my windows 10 PC for the first time. I downloaded VSC and downloaded the proper extensions be begin coding with C++. Following extensions are "C/C++" -  "C/C++ Extension Pack" - "Code Runner". My code runs correctly when I write a proper line but no matter what I attempt, I keep getting #9018. I have done everything the quick fix recommends but can't seem to find a proper solution. If anyone can help me understand what I must do, you would be a life saver as this is part of my uni work.
I see multiple attmepts on trying to solve this issue and the person attempting to help says to provide a "Log Diagnostics command".
I have no idea what that is and will also require help with that if possible.
![Capture](https://user-images.githubusercontent.com/114124339/191646954-a8e23318-628b-46da-a9be-ba0a0d1e021e.PNG)

**Comment by sean-mcmanus** [comment]
View->Command Palette->C/C++: Log Diagnostics is the command to run.

You should remove all the system headers from your includePath setting in c_cpp_properties.json and make sure your compilerPath setting is set to a valid compiler.
